packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -384,6 +384,7 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
 
   final RouteSettings settings;
 
+  /// The closest instance of this class that encloses the given context.
   static ModalRoute of(BuildContext context) {
```

Comment:
technically the routes don't enclose anything, they're off in a different world. It's the route for the closest ModalScope, but ModalScopes are an implementation detail. Dunno how to explain that here though...